packages/elements-core/src/components/Docs/HttpOperation/Body.tsx

```diff
@@ -1,20 +1,28 @@
 import { JsonSchemaViewer } from '@stoplight/json-schema-viewer';
-import { Box, Select } from '@stoplight/mosaic';
+import { Flex, Select, VStack } from '@stoplight/mosaic';
 import { IHttpOperationRequestBody } from '@stoplight/types';
 import * as React from 'react';
 
 import { useInlineRefResolver } from '../../../context/InlineRefResolver';
 import { isJSONSchema } from '../../../utils/guards';
 import { getOriginalObject } from '../../../utils/ref-resolving/resolvedObject';
 import { MarkdownViewer } from '../../MarkdownViewer';
-import { SubSectionPanel } from '../Sections';
+import { SectionSubtitle } from '../Sections';
 
 export interface BodyProps {
   body: IHttpOperationRequestBody;
   onChange: (requestBodyIndex: number) => void;
 }
 
-export const Body = ({ body: { contents = [], description }, onChange }: BodyProps) => {
+export const isBodyEmpty = (body?: BodyProps['body']) => {
+  if (!body) return false;
```

Comment:
So not having body means that body is *not* empty? 🤔 Kind of counterintuitive 😅

Reply:
lol, I'm even confused myself just reading this again - will look at it after my meetings 😅